test/toPercent.spec.ts

```diff
@@ -0,0 +1,15 @@
+import toPercent from '../src/toPercent';
+
+describe('toPercent', () => {
```

Comment:
I will suggest to include tests to cover the usage of `minPrecision` and `maxPrecision`.